app/views/unattended/provisioning_templates/PXEGrub/pxegrub_default_local_boot.erb

```diff
@@ -4,8 +4,12 @@ name: PXEGrub default local boot
 model: ProvisioningTemplate
 %>
 <%# Used to boot provisioned hosts, do not associate or change the name. %>
-
-default=0
+<%
+  # Grub1 only supports numeric default statement, this array allows conversion to number
+  items = ["local", "local_chain_hd0"]
+  default_item = items.index(global_setting("default_pxe_item_global", "local")) || 0
```

Comment:
Would this actually work? I see in https://github.com/theforeman/foreman/blob/bcddd48bdca1d6a90c651e50ebd33f1fab228b46/app/views/unattended/provisioning_templates/snippet/_pxegrub_chainload.erb#L10-L15 we're generating multiple items based on paths, so I geuss `local_chain_hd0` would not be at the index 1

Reply:
Good observation, this is problem only for Grub1. I suggest to fill the array with reasonable values, users can easily break this but unfortunately numbers are the only way in Grub1.